users/views.py

```diff
@@ -1,3 +1,78 @@
-from django.shortcuts import render
+import  json, re
+import  bcrypt
+import  jwt
+
+from    django.http                 import  JsonResponse
+from    django.views                import  View
+from    boomdoggy.settings          import  SECRET_KEY
+
+from    .models                     import  Users 
+
+
+
+class SignUp(View) : 
+    def post(self, request) : 
+        data = json.loads(request.body)
+
+        #비밀번호 암호화
+        hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
+        
+        #이메일, 비밀번호 유효성
+        REGEX_EMAIL         = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        REGEX_PASSWORD      = '^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,16}$'
+
+        try : 
+            if not (data['first_name'] and data['last_name']) : 
+                return JsonResponse({'messages' : 'PLEASE_INSERT_YOUR_NAME'}, status = 400)
```

Comment:
```suggestion
                return JsonResponse({'messages' : 'PLEASE_INSERT_YOUR_NAME'}, status = 400)
```
이 메세지를 프론트가 그대로 유저에게 전달하는게 아니라면 에러 메세지는 간단히 코드화 해주세요

Reply:
수정 반영 했습니다!